content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -87,6 +90,10 @@ Instead of creating new instances using PlatformImpl, you should always inject C
 private InventoryApi inventoryApi;
 ```
 
+##### Quick link widget removed from the list of available widgets
+
+As of release 10.14, the "Quick link" widget can no longer be found in the list of all available widgets and should no longer be used in the dashboard. Instead, we recommend you to use the "Help and service" widget.
```

Comment:
should no longer be used in the dashboard > what does this exactly mean? Should or can´t? We should clearly state the impact of this change. And what happens with the quick link widgets in my existing dashboards? Can I still use them? Will they be gone after the widget has been removed from the code base?

Reply:
Customers that might use the widget in their dashboard currently can still see and use the widget. But they need to drop the "Quick link" widget until 10.16 and use "Help and service" instead. 
Customers that are not using the "Quick link" widget currently can't use it, because it does not appear in the list of widgets anymore. From 10.16 onwards the "Quick link" widget is not working anymore.

Reply:
technically these changes have nothing to do with the SDK